client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -1,12 +1,13 @@
 /** @format */
+/* eslint-disable react/prefer-es6-class */
```

Comment:
This PR removes the only `mixin` right?
I think that means we can update the `createReactClass` call to a plain es6 class

Reply:
What @samouri said. Probably easiest to just run the file through the codemod 🙂

Reply:
This is a bit of a hangover from when I originally opened this. That was before the infinite scroll mixin was replaced & removed from this file. I do have a PR open #23695 (somethings gone awry with the rebasing so all commits are shown there).

It does make sense at this point to just apply that commit to this PR though :)

Reply:
> Probably easiest to just run the file through the codemod

I thought that but I don't think it picks up cases of `createReactClass` - I'm a little rusty on that front though so might have just been doing something wrong.

Reply:
No, you're right. You'd need to change it back to `createClass` (and `import { createClass } from 'react';`) first in order for the codemod to pick it up.

Reply:
We can drop the `/* eslint-disable react/prefer-es6-class */` line now, can't we? 🎉